src/saves/PlayerJsonSchema.ts

```diff
@@ -1,6 +1,20 @@
 import { z } from 'zod'
 import type { Player } from '../types/Synergism'
 import { playerSchema } from './PlayerSchema'
+import type { CorruptionLoadout, Corruptions } from '../Corruptions'
+
+const convertArrayToCorruption = (array: number[]): Corruptions => {
```

Comment:
I'm pretty sure this is duplicated

Reply:
There's some weird differences in indexing between the two (one is shifted by 2, for reasons I can't explain). The other one is definitely a duplicate though so I removed it in commit

Reply:
please leave comment